src/components/FormSelectList.vue

```diff
@@ -148,6 +148,7 @@
         this.filter = filter;
         this.optionsFromDataSource();
       },
+      //deprecated
```

Comment:
@marcoAntonioNina Should we remove this method since it is no longer in use?

Reply:
can be removed, I made it obsolete if used elsewhere. I am deleting it.